ballbot:
  cost_params:
    sigma:
      h: 2.0
      s: 0.66
      r: 0.33

    q:
      obs: 0.25
      goal: 0.25
      wind: 0.0

